projects/batfish/src/main/antlr4/org/batfish/grammar/flatjuniper/FlatJuniper_security.g4

```diff
@@ -512,6 +513,16 @@ s_security
    )
 ;
 
+se_auth_key_chain
+:
+   AUTHENTICATION_KEY_CHAINS KEY_CHAIN name = string
```

Comment:
Is this supposed to be `AUTHENTICATION_KEY_CHAINS` (plural) or `AUTHENTICATION_KEY_CHAIN` (singular)? Either way, the rule should be renamed to `se_authentication_key_chains` or `se_authentication_key_chain` accordingly.